src/main/java/it/gov/pagopa/pu/classification/model/AssessmentsDetail.java

```diff
@@ -43,4 +44,12 @@ public class AssessmentsDetail extends BaseEntity implements Serializable {
   @NotNull
   private boolean amountSubmitted = true;
   private Long receiptId;
+  private String officeDescription;
+  private String sectionDescription;
+  private String assessmentDescription;
+  private Long debtPositionTypeOrgId;
```

Comment:
move this before `debtPositionTypeOrgCode` field
```suggestion
  @NotNull
  private Long debtPositionTypeOrgId;
```

Reply:
This has not been resolved

Reply:
not resolved yet